uiowa/uiowa
# Feature: fix for responsive images in WYSIWYG

Resolves #3525

# How to test

`yarn workspace uids_base gulp --development`
- Place an image in the text editor and verify that it is responsive within the text editor preview
